modules/python/python_loader.cmake

```diff
@@ -38,6 +38,8 @@ foreach(fname ${PYTHON_LOADER_FILES})
   endif()
 endforeach()
 
+file(TOUCH "${__loader_path}/cv2/py.typed")
```

Comment:
`TOUCH` is available only since CMake 3.12 more portable version is the following:
```cmake
execute_process(
        COMMAND ${CMAKE_COMMAND} -E touch "py.typed"
        WORKING_DIRECTORY "${__loader_path}/cv2"
)
```